common/warm_restart.cpp

```diff
@@ -139,6 +141,20 @@ bool WarmStart::checkWarmStart(const std::string &app_name,
     return true;
 }
 
+/*
+ * Checks if fast reboot is in progress
+ */
+bool WarmStart::checkFastReboot()
+{
+    std::string value;
+
+    auto& warmStart = getInstance();
+
+    // Check system level warm-restart config first
+    warmStart.m_stateFastRestartEnableTable->hget("system", "enable", value);
```

Comment:
@arfeigin what is the behaviour when record is not present in DB? Will it throw an exception?

Reply:
According to table.cpp implementation it will return false in not present. It might throw an exception if entry exists but value is not a string.  I used the same logic used for checkWarmStart.
https://github.com/arfeigin/sonic-swss-common/blob/f02a35a566075576bdbfdf797105b618d722878d/common/table.cpp#L98